fuzzing/contracts/contract.go

```diff
@@ -35,6 +38,17 @@ type Contract struct {
 
 	// compilation describes the compilation which contains the compiledContract.
 	compilation *types.Compilation
+
+	// propertyTestMethods are the methods that are property tests.
+	propertyTestMethods []abi.Method
+
+	// optimizationTestMethods are the methods that are optimization tests.
+	optimizationTestMethods []abi.Method
+
+	// assertionTestMethods are ALL other methods that are not property or optimization tests by default.
+	// If configured, the methods will be targeted or excluded based on the targetFunctionSignatures
+	// and excludedFunctionSignatures, respectively.
+	assertionTestMethods []abi.Method
```

Comment:
I am making them all `[]*abi.Method`. Since the `Contract` will sit in the heap somewhere until medusa finishes its campaign, we might as well point to the abi methods within the ABI instead of copying-by-value.

Reply:
nvm changed my mind lol